### DISCUSSION TITLE
Welcome to Hy Discussions!

### DISCUSSION THREAD

**Discussion by allison-casey** [hylang/hy#1995] [General]
## 👋 Welcome!
  Hey all you radical cephalopods! we’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪. The [contributor code of conduct guidelines](https://github.com/hylang/hy/blob/master/CONTRIBUTING.rst#contributor-code-of-conduct) applies here as well. 
  So check that out for the kind behavior that's expected here.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paultag** [comment]
Thanks for doing this! Neat!

**Comment by 0atman** [comment]
Amazing! :tada:

**Comment by scauligi** [comment]
Relative newcomer here, hy y'all!
I write a lot of python for personal projects (unfortunately unrelated to my PhD research) and used to play around with Racket, so this is a really fun project to work with!

**Comment by tuturto** [comment]
I was using Hy a bit some years ago and now I'm coming to see all the awesome things you all have been building since.
